packages/react-reconciler/src/__tests__/ReactSuspenseWithNoopRenderer-test.js

```diff
@@ -2860,74 +2860,6 @@ describe('ReactSuspenseWithNoopRenderer', () => {
     );
   });
 
-  // @gate experimental
-  it('supports delaying a busy spinner from disappearing', async () => {
```

Comment:
Are we sure we don't want to support this use case?

Reply:
I think the idea was that longer-term, the proper way to implement this is unmount animation (which we don't have an API for). That doesn't require you to precalculate the time and would let the spinner itself decide how much it wants to stay visible before being removed. E.g. a full spin.

Reply:
Discussed out-of-band